lib/periodicChecks.js

```diff
@@ -0,0 +1,43 @@
+const assignUnAssignedReviewers = async (pullRequest, context) => {
+  const assignees = pullRequest.assignees.map(user => user.login);
+  const reviewers = pullRequest.requested_reviewers.map(user => user.login);
+
+  let botComment = 'PTAL ';
+  const prParams = context.repo(
+    {
+      number: pullRequest.number,
+    }
+  );
+
+  const unAssignedReviewers = reviewers.filter(reviewer => {
+    if (!assignees.includes(reviewer)) {
+      botComment += `@${reviewer} `;
```

Comment:
Can we add make the comment more descriptive - something along the lines: Hey @x, @y, @z, this PR is waiting on your review, can you PTAL? Thanks!

Also, one thing which is missing is - if the PR already has changes requested tag on it, it would be better to assign the author rather than the reviewer.

Reply:
Ok, so I would suggest that we should have that check in place too - i.e. assign the PR author when change is requested - since this functionality is somewhat incomplete without it. We will need to upgrade it later but it is fine to have it now since it is an essential component for this check to be complete.

Reply:
Done. PTAL